### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by Nesffer** [AtomLinter/linter-pylint#38]
Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (C:\Users\Jimin\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:810:9)
Package.hasActivationCommands (C:\Users\Jimin\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:735:20)
<unknown> (C:\Users\Jimin\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:189:24)
Package.measure (C:\Users\Jimin\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:167:15)
```

**Comment by bilderbuchi** [comment]
Should be closed as duplicate of #34.

**Comment by langerma** [comment]
also in atom 0.200.0

**Comment by kevinsawicki** [comment]
Fixed by https://github.com/AtomLinter/linter-pylint/pull/40 and published in `0.2.2`
